run/tesla_api.py

```diff
@@ -62,8 +62,8 @@ def _execute_request(url=None, method=None, data=None, require_vehicle_online=Tr
 
             # Tesla REST Service sometimes misbehaves... this seems to be caused by an invalid/expired auth token
             # TODO: Remove auth token and retry?
-            if result['response'] is None:
-                _error("Fatal Error: Tesla REST Service returned an invalid response")
+            if result.get('response') is None:
```

Comment:
`result['response']` threw exception when `response` is not in the result. eg: `{ 'error': 'error msg'}`. Log the result in the next line would be helpful for error reporting IMO.